2364/2/parser.rb

```diff
@@ -0,0 +1,48 @@
+require 'russian_obscenity'
+require 'terminal-table'
+require_relative 'participant.rb'
+require_relative 'create_data.rb'
+require_relative 'name_check.rb'
+require_relative 'obscene_rapers.rb'
+require_relative 'top_words.rb'
+
+# Class parse for parsing data
+class Parser
+  PATH = File.expand_path('.').freeze
+  attr_reader :participants, :methods
```

Comment:
i am not sure, but i think this needs to be separated with additional spaces above